app/controllers/api/account.php

```diff
@@ -173,7 +172,7 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
 $createSession = function (string $userId, string $secret, Request $request, Response $response, Document $user, Database $dbForProject, Document $project, Locale $locale, Reader $geodb, Event $queueForEvents, Mail $queueForMails) {
 
     /** @var Utopia\Database\Document $user */
-    $userFromRequest = Authorization::skip(fn () => $dbForProject->getDocument('users', $userId));
+    $userFromRequest = $dbForProject->getAuthorization()->skip(fn () => $dbForProject->getDocument('users', $userId));
 
```

Comment:
Let's inject `authorization` directly and use that instead of `$dbForProject->getAuthorization()` now that we have a separate resource for it. Then we don't have to worry about which DB we're using, and since it's an object reference, changes will apply to all DBs